Source/cryptography/implementation/OpenSSL/Vault.cpp

```diff
@@ -88,7 +88,7 @@ namespace Netflix {
                     TRACE_L1(_T("Set label to default \"%s\""), label.c_str());
                 }
 
-PUSH_WARNING(DISABLE_WARNING_NON_STANDARD_EXTENSION_USED)
+PUSH_WARNING(DISABLE_WARNING_NON_STANDARD_EXTENSION_USED, DISABLE_WARNING_PEDANTIC)
```

Comment:
Why do we need it here? I understand the zero length array [0] above but here I assume there is no issue? or ?

Reply:
There is also a zero-length array here: https://github.com/rdkcentral/ThunderClientLibraries/blob/master/Source/cryptography/implementation/OpenSSL/Vault.cpp#L96